routes/api/users.js

```diff
@@ -171,4 +173,115 @@ router.post('/users/verify/resend-email', (req, res, next) => {
   return null;
 });
 
+
+router.post('/users/forgot-password', (req, res, next) => {
+  const { email } = req.body.user;
+  if (email === undefined) {
+    return res.status(400)
+      .json({ message: 'Email can not be undefined' });
```

Comment:
I think you should extend `Jane's` work on validation and use that as a middleware instead of performing the validation here. 
You can then use validatorJS in the middleware that  you would create

Reply:
Also, the error message `Email cannot be undefined` is not user-friendly. I think you should improve the message as well